src/bugsnag.js

```diff
@@ -410,6 +410,63 @@
     enhance(history, "replaceState", wrapBuilder(buildReplaceState));
   }
 
+  function trackWorkers() {
+    if(!getBreadcrumbSetting("autoBreadcrumbsWorkers", true)) {
+      return;
+    }
+
+    var workers = 0;
+
+    window.Worker = hijackConstructor(window.Worker, function(url) {
```

Comment:
I'm coming around the way you have it currently. Internally, we currently disable the console.log breadcrumb in development to get around the line number problem. I'm actually starting to wonder if the default value for `autoBreadcrumbsConsole` and `autoBreadcrumbsWorker` should be based on the release stage.